best-practices/README.md

```diff
@@ -1,6 +1,9 @@
 # Best Practices
 
-- [Documentation](/documentation)
+- [Documentation](/best-practices/documentation)
+- [How To - Readme](/best-practices/howto_readme.md)
+- [Licensing](/best-practices/licensing.md)
+- [Version Control](/best-practices/version_control.md)
```

Comment:
This is the best place to maintain this index (and the only place to maintain it).